cms/static/js/models/xblock_info.js

```diff
@@ -120,7 +120,11 @@ function(Backbone, _, str, ModuleUtils) {
             /**
              * True iff this xblock should display a "Contains staff only content" message.
              */
-            'staff_only_message': null
+            'staff_only_message': null,
+            /**
+             * True if this xblock contains an entrance exam module.
+             */
+            'is_entrance_exam': null
```

Comment:
This seems like a strangely specific field to add to xblock info. If every feature adds a new field this is going to get out of hand (xblock info is already bloated). Was it not possible to make this a new xblock type? Could we have a new 'override_type' or something that returns an enum where 'entrance_exam' is one of the options?

I wouldn't hold up merging for this, but just wanted to record my thoughts.